### DISCUSSION TITLE
updateassets vs updatestyles?

### DISCUSSION THREAD

**Discussion by HansF** [pretix/pretix#4338] [Q&A] [answered]
Following the [small-scale guide](https://docs.pretix.eu/en/latest/admin/installation/manual_smallscale.html#updates), I tried this command.  

(venv) $ python -m pretix updateassets
_Unknown command: 'updateassets'. Did you mean updatestyles?
Type 'python -m django help' for usage._
(venv) $ python -m pretix updatestyles
_/var/pretix/venv/lib/python3.11/site-packages/pypdf/_crypt_providers/_cryptography.py:32: CryptographyDeprecationWarning: ARC4 has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.ARC4 and will be removed from this module in 48.0.0.   from cryptography.hazmat.primitives.ciphers.algorithms import AES, ARC4_

Am I missing something? Do I need to install something else for updateassets to work?

**Comment by raphaelm** [comment, marked as answer]
Sorry, the docs is already updated for the very-soon-to-be-released pretix 2024.7, where it is renamed from updatestyles to updateassets.
